advtraj/utils/grid.py

```diff
@@ -62,9 +63,10 @@ def wrap_periodic_grid_coords(
         if c in cell_centered_coords:
             x_min -= dx / 2.0
             x_max += dx / 2.0
+        else:
+            x_max += dx
```

Comment:
I think this makes sense, there was a bug in my code before for coordinates on cell-faces, is that right?

Reply:
Yes